Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by Jaehun Lee on 2/13/24.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        var operands = CalculatorItemQueue<Double>()
+        var operators = CalculatorItemQueue<Operator>()
+        
+        let operandsArr = componentsByOperators(from: input)
+            .compactMap { Double($0) }
+        let operatorsArr = input
+            .compactMap { Operator(rawValue: $0) }
+        
+        operandsArr.forEach { operands.enqueue(element: $0) }
+        operatorsArr.forEach { operators.enqueue(element: $0) }
+        
+        return Formula(operands: operands, operators: operators)
```

Comment:
```suggestion
var operands: CalculatorItemQueue<Double> = .init()
        var operators: CalculatorItemQueue<Operator> = .init()
        
        componentsByOperators(from: input)
            .compactMap(Double.init)
            .forEach { operands.enqueue(element: $0) }
        input
            .compactMap(Operator.init)
            .forEach { operators.enqueue(element: $0) }
        
        return Formula(operands: operands, operators: operators)
```

요러면 코드량이 조금 더 줄어들 수 있겠네요 ㅎㅎ

그리고 타입을 명시해주면 컴파일 타임이 아주 조금 빨라져요

사실 componentsByOperators에서 
operator의 배열, 문자열의 배열 요렇게 나눠서 주는게 깔끔할거 같긴 한데 지금은 넘어가는거로 ㅎㅎ

Reply:
타입 추론을 할 필요가 없어지기 때문에 조금이나마 컴파일 타임이 빨라지는 것이군요 👍